src/licensedcode/data/rules/tsl_2.yml

```diff
@@ -0,0 +1,5 @@
+license_expression: tsl-2018 OR tsl-2020
+is_license_notice: yes
+relevance: 100
+ignorable_urls:
+   - https://github.com/timescale
```

Comment:
IMHO you could add:
```
referenced_filenames:
    - LICENSE-TIMESCALE
 ```
Also, does the license mention you can use either version if the data is not there?

Reply:
This should be `tsl-2020` only IMHO, as according to the text in the license added [here](https://github.com/nexB/scancode-toolkit/pull/2367/commits/8d9c2d8f4076406467b1b75588524fa859df95e8).

```
1.2 License Rights to Your Customers.  As set forth in Section 2.1 below,
   the use by Your customers of the Timescale Software as part of any Value
   Added Products or Services that You distribute will be subject to the most
   current version of this TSL Agreement.
```

Reply:
@AyanSinhaMahapatra Yeah according to 2.1, you are correct. I'ii change this.

Reply:
@itssingh gentle ping. I would like to merge this now that a new release was pushed.

Reply:
@pombredanne Do I need to do somechanges?